### DISCUSSION TITLE
My sumatra switched somehow to white on black display

### DISCUSSION THREAD

**Discussion by florinav** [sumatrapdfreader/sumatrapdf#4107] [General]
The first lines in settings are 
TextColor = #000000
	BackgroundColor = #ffffff
	SelectionColor = #f5fc0c

How do I switch back to black on white?  Thanks

**Comment by GitHubRulesOK** [comment]
that is correct black on white but you have triggered `I`nversion press the `I` key

**Reply by florinav** [reply]
Thanks, is the I key some sort of keyboard shortcut?

**Reply by GitHubRulesOK** [reply]
I  is both the old and the newer Shortcut for Invert all colors (at one time it was shifted I)

**Reply by florinav** [reply]
So I just type I on the keyboard ? There must be some CTRL, so it knows I'm not typing, but commanding

**Reply by GitHubRulesOK** [reply]
there are a  very high number of users wanted to reverse ALL colours at night !!! 
so it was made easier to push a single key ! (but it was a single key for `I`nfo anyway)
to re-assign set cmd for `i` to none and replace the command with your own choice

**Reply by florinav** [reply]
I'm sorry to insist.  I'm a casual Windows user, and I don't understand where   I must type  my I (I'm sure it's not in the word processor, though)

**Reply by GitHubRulesOK** [reply]
The way to set a key to none is described in https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts so for <kbd>i</kbd> will need to be   added to Advanced Opions in the Shortcuts section
```
Shortcuts [
	[
		Cmd = CmdNone
		Key = i
	]
]
```
to set invert colors to another key  like CTRL+i add that too it should work as soon as you save the advanced options with this replaced section
```
Shortcuts [
	[
		Cmd = CmdNone
		Key = i
	]
	[
		Cmd = CmdInvertColors
		Key = Ctrl + i
	]
]
```

**Reply by florinav** [reply]
Thank you all very much for your time. Clearly, my question was not well formulated, since the answers dealt with other issues. Meanwhile, the problem fixed itself. It seems I touched something by mistake on my old keyboard, which had this inversion strange effect. Later, I took the portable to a repair shop, and upon my return the problem had fixed itself, for some mysterious reason :)

**Reply by Hiche3** [reply]
`Shortcuts [
	[
		Cmd = CmdNone
		Key = i
	]
]`
So this will disable the inverting text shortcut completely?

**Reply by GitHubRulesOK** [reply]
@Hiche3 
they should be separate lines as shown above.

**Comment by Hiche3** [comment]
Same happened to me a couple of weeks ago. I couldn't figure out what triggered it or how to undo. Ended up deleting the configuration file to get everything working again.

Pressing "i" by itself should not trigger this. 
Would it be possible to add an option to disable this, or at least make the default ctrl + i

**Comment by Tyrben** [comment]
This should be accessible by menu, not only by shortcut.
For example: Display > Invert colors

So the User could have a feedback (for example a checked/unchecked Inverted colors in the menus)

**Reply by kjk** [reply]
It's also available via [Command Palette](https://www.sumatrapdfreader.org/docs/Command-Palette)
